src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.js

```diff
@@ -364,6 +374,24 @@ function ComposerWithSuggestions({
         [isKeyboardShown, isSmallScreenWidth, parentReportActions, report, reportActions, reportID, handleSendMessage, suggestionsRef, valueRef],
     );
 
+    const onChangeText = useCallback(
+        (commentValue) => {
+            updateComment(commentValue, true);
+
+            if (isIOSNative && syncSelectionWithOnChangeTextRef.current) {
```

Comment:
Absolutely, this is true - however, since this has negligent impact on performance, I wanted it to stay there as a defensive condition for the code to be clean & stable. Please consider a situation when something that sets this ref is introduced in the future, in which case this condition would still prevent the code from being executed and also it explicitly shows that this is only to be set on iOS native. Would you agree to leave this here, then?